src/core/make-popup.js

```diff
@@ -31,6 +31,7 @@ import { handleAutoOpen } from './utils/popup-auto-open'
 const DEFAULT_DRAWER_WIDTH = 800
 const DEFAULT_POPUP_WIDTH = 320
 const DEFAULT_POPUP_HEIGHT = 500
+const DEFAULT_POPUP_SIZE = 100
```

Comment:
Maybe it will make sense to indicate that this value is a percent (%). So you can pass the value as '75%' and not as just '75'. However, I see that for the other units HEIGHT & WIDTH, we are using pixels and we are not providing the units.

Maybe we could just add a comment in the code to specify that this 100 (DEFAULT_POPUP_SIZE) is a percentage and NOT pixels like the other ones could do. Or we could discuss as a team if we want to be able to specify the units on a further refactor.

Reply:
Since we are not using units I think we should omit it here as well. Maybe we could rename this constant to `DEFAULT_POPUP_SIZE_PERCENT`?

Reply:
I think renaming the constant is a good approach, it makes it clear what unit is using